### PR TITLE
Fix reload theme breaking BG sizing (needs testing with animated BGs)

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Fixes https://github.com/AttorneyOnline/AO2-Client/issues/171.

### CODE REVIEW

File: src/aoscene.cpp [hunk 1]
```diff
@@ -43,9 +43,12 @@ void AOScene::set_image(QString p_image)
 
     m_movie->setScaledSize(QSize(n_w, n_h));
     this->resize(m_movie->scaledSize());
-    this->setMovie(m_movie);
+    if (!file_exists(background_path) || background_path != last_image)
```
Comment (oldmud0) [inline src/aoscene.cpp:46]: suggest extracting variable `restart`

Comment (Crystalwarrior) [reply src/aoscene.cpp:46]: @oldmud0 instructions unclear, rebased entire DRO codebase into AO2-Client

Comment (Crystalwarrior) [reply src/aoscene.cpp:46]: seriously tho what do you mean lol

Comment (oldmud0) [reply src/aoscene.cpp:46]: the logic is a copypaste from the previous check

**Pull request merged**